src/camvidlog2/ha_mqtt_client.py

```
@@ -0,0 +1,113 @@
+        self,
+        exc_type: type | None,
+        exc_val: BaseException | None,
+        exc_tb: Any | None,
+    ) -> None:
+        self.client.loop_stop()
+        self.client.disconnect()
+        self._connect_event = None
+
+    def _on_connect(
+        self,
+        client: mqtt.Client,
```

Ruff (0.11.9)

Unused method argument: `client`

(ARG002)